docs/clusterpools.md

```diff
@@ -147,3 +147,81 @@ spec:
 ```
 
 **Note** When you use installConfigSecretTemplate you will most likely want to disable MachinePools, so that Hive does not reconcile away from the machine config specified in install-config.yaml
+
+## Time-based scaling of Cluster Pool
+
+You can use kubenetes cron jobs to scale clusterpools as per a defined schedule. Following are the sample yaml configurations to scale a clusterpool to size 0 at 23:45 (11:45 PM) every Friday. 
+
+```yaml
```

Comment:
split these up into 2 sections, 

one set of yaml is for setting up the rbac,
and the next set is examples of how to scale up and down based on time. <== for this one let's pick up a real world example, like 6 am (some timezone) everyday run scale up to X and 8 pm same day scale down to Y. I think an example like that conveys more...